File: JavaScript/bee 2456.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split(' ');

function bubbleSortCrescente(arr){
    for(let i = 0; i < 5; i++){
        for(let j = 0; j < 5; j++){
            if(arr[j] > arr[j+1]){
                let temp = arr[j]
                arr[j] = arr[j+1]
                arr[j+1] = temp
            }
        }
    }
    return arr
}

function bubbleSortDecrescente(arr){
    for(let i = 0; i < 5; i++){
        for(let j = 0; j < 5; j++){
            if(arr[j] < arr[j+1]){
                let temp = arr[j]
                arr[j] = arr[j+1]
                arr[j+1] = temp
            }
        }
    }
    return arr
}

let arrCartas = lines.map(Number)
let arrCartasCrescente = bubbleSortCrescente(lines.map(Number))
let arrCartasDecrescente = bubbleSortDecrescente(lines.map(Number))

let qtdCrescente = 0
let qtdDecrescente = 0

for(let i = 0; i < 5; i++){
    if(arrCartas[i] == arrCartasCrescente[i]){
        qtdCrescente++
    }
    if(arrCartas[i] == arrCartasDecrescente[i]){
        qtdDecrescente++
    }
}   

if(qtdCrescente == 5){
    console.log('C')
} else if(qtdDecrescente == 5){
    console.log('D')
} else{
    console.log('N')
}

File: C++/bee 2456.cpp
#include <iostream>
#include <iomanip>
#include <stdio.h>

using namespace std;

int main()
{
    int arr[5];
    int n;
    int qtdCrescente = 0;
    int qtdDecrescente = 0;

    for(int i = 0; i < 5; i++){
        cin >> n;
        arr[i] = n;
    }
    for(int i = 0; i < 4; i++){
        if(arr[i] < arr[i+1]){
            qtdCrescente++;
        }
        if(arr[i] > arr[i+1]){
            qtdDecrescente++;
        }
    }
    if(qtdCrescente == 4){
        cout << "C" << endl;
    } else if(qtdDecrescente == 4){
        cout << "D" << endl;
    } else{
        cout << "N" << endl;
    }
}
